frontend/app/src/test/java/edu/uw/cs403/plantmap/ExampleUnitTest.kt

```diff
@@ -12,6 +12,6 @@ import org.junit.Assert.*
 class ExampleUnitTest {
     @Test
     fun addition_isCorrect() {
-        assertEquals(4, 2 + 2)
+        assertEquals(3, 2 + 2)
```

Comment:
Here's where the test file is failing. This is incorrect and should be 4! I'll fix it so the tests can run again.